### PR TITLE
Add warning, when update-check is enabled, but no stable release image is used

### PR DESCRIPTION
**Pull request by casperklein**
# Description

This PR adds a warning, when update-check is enabled, but no stable release image is used.

Related: #3666

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (non-breaking change that does improve existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (README.md or the documentation under `docs/`)
- [ ] If necessary I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] **I have added information about changes made in this PR to `CHANGELOG.md`**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ All notable changes to this project will be documented in this file. The format
 
 > **Note**: Changes and additions listed here are contained in the `:edge` image tag. These changes may not be as stable as released changes.
 
+- **Internal:**
+  - Add warning, when update-check is enabled, but no stable release image is used ([#3684](https://github.com/docker-mailserver/docker-mailserver/pull/3684))
+
```
Comment (polarathene) [inline CHANGELOG.md:11]: Should be moved into the "Added" section?

**Review by polarathene — DISMISSED** [review]
Nice improvement! 😁

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ All notable changes to this project will be documented in this file. The format
 
 > **Note**: Changes and additions listed here are contained in the `:edge` image tag. These changes may not be as stable as released changes.
 
+- **Internal:**
```
Comment (georglauterbach) [inline CHANGELOG.md:9]: This change actually does not look internal to me 😆

**Review by georglauterbach — APPROVED** [review]
LGTM 👍🏼

File: CHANGELOG.md (hunk 1 above)
Comment (georglauterbach) [reply CHANGELOG.md:11]: You literally beat me to this :D

Comment (georglauterbach) [reply CHANGELOG.md:11]: I will adjust and then we can merge.

Comment (georglauterbach) [reply CHANGELOG.md]: Done.

**Review by georglauterbach — APPROVED** [review]

**Pull request merged**